adblock-lean

```diff
@@ -437,15 +439,33 @@ generate_preprocessed_blocklist_file_parts()
 
 			rm -f /var/run/adblock-lean/blocklist_part_size_B /var/run/adblock-lean/blocklist_part_line_count
 
+			if read -r rogue_element < /var/run/adblock-lean/rogue_element
```

Comment:
But shouldn't we verify that a download completed before trusting the result of the rogue element check (since a partial download could presumably itself result in a rogue element)?

Reply:
Should be fixed now, with rogue el. check after download completed check.